utilities/OnYourDataIngestionAPI.ps1

```diff
@@ -2,71 +2,71 @@
 
 # Azure Open AI configuration
 
-$AzureOpenAIEndpoint = "<resource-name>.openai.azure.com"
-$EmbeddingDeploymentName = "embedding"
+$AzureOpenAIEndpoint = "knop-swedencentral-azopenai.openai.azure.com"
+$EmbeddingDeploymentName = "hub-embedding-text-embedding-ada-002"
 
 # Azure AI search configuraton
 
-$AzureAiSearchEndpoint = "<resource-name>.search.windows.net"
+$AzureAiSearchEndpoint = "knop-swedencentral-azaisearch.search.windows.net"
```

Comment:
I'd revert this back to how you had it originally without specific endpoint names.